inventory/group_vars/all.yml

```diff
@@ -9,7 +9,7 @@ namespace_prefix: "<YOUR_NAME>"
 
 # When the ocp_templates gets moved to cop land we can merge the two raws below
 openshift_templates_raw: "https://raw.githubusercontent.com/rht-labs/openshift-templates"
-openshift_templates_raw_version_tag: "v1.4.9"
+openshift_templates_raw_version_tag: "v1.4.17"
 cop_quickstarts: "https://github.com/redhat-cop/containers-quickstarts.git"
 cop_quickstarts_raw: "https://raw.githubusercontent.com/redhat-cop/containers-quickstarts"
-cop_quickstarts_raw_version_tag: "v1.9"
+cop_quickstarts_raw_version_tag: "master"
```

Comment:
I would prefer that we stick with a release. I can create a new release if there's features missing in the latest release but available in `master`.

Reply:
right! the latest version is good enough for this :)